packages/connectivity_plus/connectivity_plus/android/build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:8.3.1'
+        classpath 'com.android.tools.build:gradle:8.3.2'
```

Comment:
This doesn't make any sense, because 8.3.1 also a valid version if you check releases of the AGP: https://mvnrepository.com/artifact/com.android.tools.build/gradle

<img width="600" alt="Screenshot 2024-12-08 at 00 52 21" src="https://github.com/user-attachments/assets/1a511d75-e2de-43b0-a881-a12287a5b63d">